helm-v3/005-test.md

```diff
@@ -0,0 +1,45 @@
+# Test Command
+
+## Overview
+
+`helm test` gives a chart author the ability to define actions on a helm release which validate that the chart is running as intended. This is useful for ensuring that a `helm install` or `helm upgrade` created/modified Kubernetes resources successfully and that the consequential `helm release` is functional.
+* A **test** is defined as a command and some arguments executed from a container running in a Kubernetes Pod. If the command completes inside the container with an exit code of 0, it is successful. All other exit codes indicate the test failed.
```

Comment:
I think we should support Jobs as well as Pods (should match whatever is supported by helm 3 hooks). See also https://github.com/helm/community/pull/40#discussion_r263521243.